sdk/identity/.dict.txt

```diff
@@ -9,5 +9,6 @@ imds
 managedidentity
 msal
 replacen
+structs
```

Comment:
Now that you call this out, how used in the document, maybe you shouldn't abbreviate and just say "structures".